CHANGELOG.md

```diff
@@ -85,7 +89,8 @@ Improve CI/CD flow:
 - New "tags" parameter to operations object in Swagger format.
 - Add changelog and Makefile.
 
-[unreleased]: https://github.com/brainn-co/xcribe/compare/v0.7.3...master
+[unreleased]: https://github.com/brainn-co/xcribe/compare/v0.7.4...master
+[0.7.4]: https://github.com/brainn-co/xcribe/compare/0.7.3...0.7.4
```

Comment:
The tag is v0.7.3, isn't it? 

```suggestion
[0.7.4]: https://github.com/brainn-co/xcribe/compare/v0.7.3...v0.7.4
```

Reply:
Yes, I fixed the release script to be in the correct format. 

@antedeguemon Done.